typescript/src/policies/schema.ts

```diff
@@ -107,6 +107,20 @@ export const EvmNetworkCriterionSchema = z.object({
 });
 export type EvmNetworkCriterion = z.infer<typeof EvmNetworkCriterionSchema>;
 
+/**
+ * Schema for EVM message criterions
+ */
+export const EvmMessageCriterionSchema = z.object({
+  /** The type of criterion, must be "evmMessage" for EVM message-based rules. */
+  type: z.literal("evmMessage"),
+  /**
+   * A regular expression the message is matched against.
+   * Accepts valid regular expression syntax described by [RE2](https://github.com/google/re2/wiki/Syntax).
+   */
+  match: z.string(),
```

Comment:
wonder if there's a regex FOR a regex... 🤔 
i'd at least put a `.min(1)` on this